adhoc-openshift-pv.yml

```diff
@@ -0,0 +1,57 @@
+---
+# This playbook is to create a persistent volume for openshift CI namespace
+# Example to run the playbook
+# `ansible-playbook -e "env_prefix=ocp-prod" -e "pv_size=10Gi" -e
+# "cico_project_name=new_fancy_project" adhoc-openshift-pv.yml -i inventory`
+
+
+- hosts: ocp-ci-management
+  become: true
+
+  tasks:
+    - name: Generate a UUID
+      set_fact:
+        pv_uuid: "{{ (cico_project_name + pv_claimref|default('noclaimref', true)) | to_uuid }}"
+
+    - name: UUID Generated
+      debug:
+        var: pv_uuid
+
+    - name: Make a pv name
+      set_fact:
+        pv_name: "pv-{{ pv_size | lower }}-{{ pv_uuid }}"
+
+    - name: UUID Generated
+      debug:
+        var: pv_name
+
+    - name: See if the PV already exists
+      command: "oc get pv/{{ pv_name }}"
```

Comment:
That will not work : ansible doesn't know which user to become at this stage (nothing declared) so oc command will fail because no auth for openshift, nor full path to oc binary either